plugin.video.ahltv/resources/lib/utils.py

```diff
@@ -0,0 +1,105 @@
+import pytz, time
+import os
+import calendar
+from datetime import date, datetime, timedelta
+from cStringIO import StringIO
```

Comment:
Not compatible with Python 3. You should better use `io.BytesIO` class

Reply:
Updated and tested against Kodi nightly